keras_cv/visualization/colors.py

```diff
@@ -0,0 +1,151 @@
+colors = {
+    # X11 color to RGB tuple.
```

Comment:
If taken from PIL we need to comply w/ their license:

https://raw.githubusercontent.com/python-pillow/Pillow/6deac9e3a23caffbfdd75c00d3f0a1cd36cdbd5d/LICENSE


We may want to prefer generating our own colors from CSS colors.

Reply:
yup, nice refactor suggest done :)